common/sass/fonts/icons/icons.scss

```diff
@@ -0,0 +1,44 @@
+/*------------------------
+	font icons
+-------------------------*/
+
+.nc-icon.nc-logo-facebook:before {
+  content: "\ea02";
+}
+
+.nc-icon.nc-logo-reddit:before {
+  content: "\ea03";
+}
+
+.nc-icon.nc-logo-github:before {
+  content: "\ea04";
+}
+
+.nc-icon.nc-logo-twitter:before {
+  content: "\ea05";
+}
+
+.nc-icon.nc-logo-linkedin:before {
+  content: "\ea06";
+}
+
+.nc-icon.nc-logo-slack:before {
+  content: "\ea07";
+}
+
+.nc-icon.nc-logo-medium:before {
+  content: "\ea08";
+}
+
+.nc-icon.nc-alert-circle-que:before {
+  content: "\ea0f";
+}
+
+.nc-icon.nc-eye-19:before {
+  content: "\ea10";
+}
+
+.nc-icon.nc-eye-ban-20:before {
```

Comment:
What are the numbers for here? Can they just be `nc-eye` and `nc-eye-closed` or something?

Reply:
They correspond to the icon names in the [nucleo](https://nucleoapp.com/) icon-font, I'll go back to using font-awesome and change the names accordingly